src/scripts/main.js

```diff
@@ -354,7 +354,47 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.getElementsByClassName('dashboard')[0];
 
-// write your code here
+for (const person of people) {
+  const tableRow = document.createElement('tr');
+
+  const personName = document.createElement('td');
+
+  personName.textContent = person.name;
+
+  const personGender = document.createElement('td');
+
+  if (person.sex === 'm') {
+    personGender.textContent = 'Male';
+  } else {
+    personGender.textContent = 'Female';
+  }
+
+  const personBorn = document.createElement('td');
+
+  personBorn.textContent = person.born;
+
+  const personDied = document.createElement('td');
+
+  personDied.textContent = person.died;
+
+  const personAge = document.createElement('td');
+
+  personAge.textContent = person.died - person.born;
+
+  const personCentury = document.createElement('td');
+
+  personCentury.textContent = Math.ceil(person.died / 100);
```

Comment:
The calculation of the century using Math.ceil(person.died / 100) may not be accurate for years ending in 00. For example, the year 1900 should be in the 19th century, not the 20th. Consider using Math.floor((person.died - 1) / 100) + 1 to correctly calculate the century.